src/oryx/install.sh

```diff
@@ -241,6 +241,7 @@ if [[ "${PINNED_SDK_VERSION}" != "" ]]; then
     MAJOR_MINOR_PATCH1_VERSION=${PINNED_SDK_VERSION%??}
     rm -rf /usr/share/dotnet/shared/Microsoft.NETCore.App/$MAJOR_MINOR_PATCH1_VERSION
```

Comment:
Can we bump the patch version for this Feature?
https://github.com/devcontainers/features/blob/0d4771c789f0b007b538aaf0922d96f00d574f9c/src/oryx/devcontainer-feature.json#L3

Reply:
done